nsv13/code/__HELPERS/priority_announce.dm

```diff
@@ -17,6 +17,5 @@
 		if(AR.redalert == state)
```

Comment:
above here it is checking all the teleportlocs instead I believe it should only check the station_areas which is its own list and should be "SMALLER" (The longer term thing would have to pass which list it should check down the line)